toolkits/global/packages/global-forms/README.md

```diff
@@ -13,3 +17,265 @@ The `global-forms` component currently uses the `DEFAULT` brand only.
 // Include this with your other components
 @import '@springernature/global-forms/scss/50-components/forms';
 ```
+
+Then you will need to register the handlebars partials in the `/view` folder. 
```

Comment:
You can, in a later version.

Reply:
> Could we include a js/java example here on how best to register the partials?

This sounds like an implementation detail that would be better documented further up in the Elements docs site, or in the application docs itself, or even somewhere in the Engineering Enablement rather than repetition per component docs page.

Reply:
👍